tools/azsdk-cli/Azure.Sdk.Tools.Cli.Tests/Helpers/DotNetPackageInfoHelperTests.cs

```diff
@@ -58,7 +59,7 @@ public async Task FindSamplesDirectory_WithSampleFiles_ReturnsSamplesDirectory()
         // Create non-sample files
         CreateTestFile(packagePath, "tests/unit/other.cs", "namespace Test; public class NotASample { }");
 
-        var helper = new DotnetLanguageService(processHelper, powershellHelper, gitHelper, new TestLogger<DotnetLanguageService>(), commonValidationHelpers);
+        var helper = new DotnetLanguageService(processHelper, powershellHelper, gitHelper, new TestLogger<DotnetLanguageService>(), commonValidationHelpers, Mock.Of<IFileHelper>());
```

Comment:
Should we refactor the tests as well? DotNetPackageInfoHelper does not exist anymore.